ext/oci8/tests/reflection1.phpt

```diff
@@ -241,13 +240,6 @@ Function [ <internal%s> function oci_free_statement ] {
   }
 }
 
-Function [ <internal%s> function oci_internal_debug ] {
-
-  - Parameters [1] {
-    Parameter #0 [ <required> $mode ]
-  }
-}
-
```

Comment:
I see that there are a few functions (e.g. `oci_parse()`) for which reflection info is not up-to-date anymore since stubs were added. Or doesn't this test currently fail?

Reply:
I believe the test failed before the changes too. Should i remove this test completely, based on your other comment?

Reply:
I've removed a few such tests myself, so I think this can be deleted as well.

Reply:
Yes, can be dropped. Especially as the `export()` method used here doesn't even exist anymore in PHP 8...